src/xkb_bell.c

```diff
@@ -76,8 +79,10 @@ void initialize_xkb(void) {
 		return;
 	}
 
-	if (! XkbOpenDisplay(DisplayString(dpy), &xkb_base_event_type, &ir,
-	    NULL, NULL, &reason) ) {
+	xkb_disp = XkbOpenDisplay(DisplayString(dpy), &xkb_base_event_type, &ir,
+		NULL, NULL, &reason);
+
+	if (!xkb_disp) {
```

Comment:
Well the connection (in terms of returned pointer) was and is not used anywhere, According to https://linux.die.net/man/3/xkbopendisplay, this cloud maybe be refactored?